index.ftd

```diff
@@ -1706,41 +1752,604 @@ width: fill-container
+-- component post-card-ui:
+caption title:
+string timestamp:
+ftd.image-src image: 
+string url:
+body body:
+optional common.author-meta author:
+common.tags-list list tags:
+
+-- ftd.column:
+width: fill-container
+
+-- ftd.row:
+width: fill-container
+spacing.fixed.px: 60
+
+-- ftd.column:
+
+-- ftd.row:
+width: fill-container
+
+-- topic: $obj.title
+$loop$: $post-card-ui.tags as $obj
+tag-link: $obj.tag-link
+active: true
+
+-- ftd.text: $post-card-ui.timestamp
+color: $inherited.colors.text
+role: $inherited.types.label-small
+padding-horizontal.px: 12
+padding-vertical.px: 5
+
+-- end: ftd.row
+
+-- ftd.text: $post-card-ui.title
+color: $inherited.colors.text
+role: $inherited.types.heading-small
+margin-top.px: 24
+
+-- ftd.text:
+color: $inherited.colors.text
+role: $inherited.types.copy-regular
+margin-top.px: 12
+
+$post-card-ui.body
+
+-- end: ftd.column
+
+-- ftd.image:
+src: $post-card-ui.image
+height.fixed.px: 200
+width.fixed.px: 300
+
+-- end: ftd.row
+
+-- end: ftd.column
+
+-- end: post-card-ui
+
+
+
+
+
+
+
+
+
+
+-- component featured-post-card:
+caption title:
+optional body body:
+optional string url:
+optional string timestamp:
+optional ftd.image-src image:
+common.author-meta author:
+optional string extra-text:
+optional string cta-primary: 
+optional string cta-primary-url:
+optional string cta-secondary: 
+optional string cta-secondary-url:
+
+-- ftd.column:
+width: fill-container
+padding-vertical.px: 120
+
+-- ftd.row:
+width: fill-container
+border-left-width.px: 19
+border-color: $inherited.colors.accent.primary
+spacing: space-between
+
+-- ftd.column:
+width.fixed.px: 445
+margin-left.px: 24
+align-self: center
+
+-- ftd.text: $featured-post-card.title
+color: $inherited.colors.text
+role: $inherited.types.heading-large
+
+-- ftd.text:
+color: $inherited.colors.text
+role: $inherited.types.copy-regular
+margin-top.px: 24
+
+$featured-post-card.body
+
+-- ftd.row:
+padding-horizontal.px: 20
+padding-vertical.px: 12
+spacing.fixed.px: 8
+background.solid: $inherited.colors.cta-primary.base
+align-content: center
+margin-top.px: 24
+
+-- ftd.text: Button
+color: $inherited.colors.cta-primary.text
+role: $inherited.types.copy-regular
+link: /
+
+-- ftd.image:
+src: $assets.files.assets.tick.svg
+
+-- end: ftd.row
+
+-- end: ftd.column
+
+-- ftd.image:
+width.fixed.percent: 55
+src: $featured-post-card.image
+align-self: end
+
+-- end: ftd.row
+
+-- end: ftd.column
+
+-- end: featured-post-card
+
+
+
+
+
+
+
+
+
+
+-- component featured-post:
+common.post-meta post-data:
+optional string cta-primary: 
+optional string cta-primary-url:
+optional string cta-secondary: 
+optional string cta-secondary-url:
+optional string extra-text:
+
+-- featured-post-card: $featured-post.post-data.title
+timestamp: $featured-post.post-data.published-on
+author: $featured-post.post-data.author
+body: $featured-post.post-data.body
+image: $featured-post.post-data.post-image
+extra-text: $featured-post.extra-text
```

Comment:
```suggestion
```

Reply:
This is the property exists on priyanka's UI

Reply:
@saurabh-lohiya still I am not convinced why its is called as `extra-ext` can it be called as `info`?

Reply:
We can call that as info but priyanka will have to change that first

Reply:
@saurabh-lohiya change it with `info` and add comment to Priyanka's thread.